### DISCUSSION TITLE
Open Source ERP support site built with doks

### DISCUSSION THREAD

**Discussion by mshiner** [thuliteio/doks#138] [Showcase]
Still WIP but here ya go

![uzERP support page](https://user-images.githubusercontent.com/16088458/108259151-b0129280-7158-11eb-9ff3-f4641902378b.png)

uzERP open source business software, including accounting, stock, manufacturing and more.

https://support.uzerp.com/

Best regards
M

**Comment by h-enk** [comment]
Hello Martyn,

Nicely done! I noticed a few issues:

1. The next and previous page buttons on the [documentation pages](https://support.uzerp.com/docs/getting-started/introduction/) are not in the correct order — make sure you have the weight (in the frontmatter of a a documentation page) set right (i.e. in the right order)
2. The articles index pages don't show quite as intended*:
    - [Articles for developers and power users](https://support.uzerp.com/articles/)
    - [Developer Focused Articles](https://support.uzerp.com/articles/technical/)
    - [Tutorials](https://support.uzerp.com/articles/tutorial/)
3. Also, the header doesn't show as intended on small screens**

*Add to `./assets/scss/common/_global.scss`:

```scss
.showcase.list .content {
  padding-top: 1rem;
  padding-bottom: 3rem;
}
```
**In `./layouts/partials/header/header.html` disable /hide the search form like so:

```html
      ..
      <div class="break order-6 d-md-none"></div>
      <!--
      <form class="navbar-form flex-grow-1 order-7 order-md-3">
        <input id="userinput" class="form-control is-search" type="search" placeholder="Search docs..." aria-label="Search docs..." autocomplete="off">
        <div id="suggestions" class="shadow bg-white rounded"></div>
      </form>
      -->
    </div>
  </div>
</header>
```

If you need extra help, let me know.

After the fixes 😉 are you OK with me adding the uzERP Support website to the Doks showcase (guess so, but I have to ask)?

**Reply by mshiner** [reply]
Henk

Again - that you so much for the help. I've added the search. Will sort the weights thing in due course.

BTW the _global.scss file was as you posted above.

M

**Reply by mshiner** [reply]
Henk

Also..... totally happy for you to add our site to the Doks showcase.

Martyn
